.gitignore

```diff
@@ -11,3 +11,4 @@
 /tmp-embed/
 /RPMS/
 /crc
+.idea
```

Comment:
This should be part of your local git global config instead here.

Reply:
However, some files we can add to gitignore to prevent them littering the repo.

Reply:
I find keeping this in `.gitignore` is helpful for developing on Goland.

Reply:
You can add `.idea` to `~/.config/git/ignore`, it's applied to all git repositories, this way you don't need to manually update .gitignore in each project you contribute to.